x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/response_actions_log.test.tsx

```diff
@@ -570,6 +499,289 @@ describe('Response actions history', () => {
       userEvent.click(getByTestId('superDatePickerCommonlyUsed_Last_15 minutes'));
       expect(startDatePopoverButton).toHaveTextContent('Last 15 minutes');
     });
+
+    describe('`get-file` action', () => {
+      it('should contain download link in expanded row for `get-file` action WITH file operation permission', async () => {
+        useUserPrivilegesMock.mockReturnValue({
+          endpointPrivileges: getEndpointAuthzInitialStateMock({
+            canWriteExecuteOperations: false,
+          }),
+        });
+        mockUseGetEndpointActionList = {
+          ...getBaseMockedActionList(),
+          data: await getActionListMock({ actionCount: 1, commands: ['get-file'] }),
+        };
+
+        mockUseGetFileInfo = {
+          isFetching: false,
+          error: null,
+          data: apiMocks.responseProvider.fileInfo(),
+        };
+
+        render();
+
+        const { getByTestId } = renderResult;
+        const expandButton = getByTestId(`${testPrefix}-expand-button`);
+        userEvent.click(expandButton);
+
+        await waitFor(() => {
+          expect(apiMocks.responseProvider.fileInfo).toHaveBeenCalled();
+        });
+
+        const downloadLink = getByTestId(`${testPrefix}-getFileDownloadLink`);
+        expect(downloadLink).toBeTruthy();
+        expect(downloadLink.textContent).toEqual(
+          'Click here to download(ZIP file passcode: elastic).Files are periodically deleted to clear storage space. Download and save file locally if needed.'
+        );
+      });
+
+      it('should show file unavailable for download for `get-file` action WITH file operation permission when file is deleted', async () => {
```

Comment:
This is testing the Download link component and not how its being used from response actions log. I think this test is already covered in that component's list of tests, so IMO, you don't need to duplicated it here.

Reply:
7a1f87b1bee40512a7d11b44ad16cd3d781748c6